src/pages/studyView/StudyViewUtils.tsx

```diff
@@ -231,8 +234,104 @@ export function getVirtualStudyDescription(
 export function isFiltered(filter: StudyViewFilter) {
     return !(_.isEmpty(filter) || (
         _.isEmpty(filter.clinicalDataEqualityFilters) &&
+        _.isEmpty(filter.clinicalDataIntervalFilters) &&
```

Comment:
if this this collection is likely to change, maybe we write something that iterates and checks for emptiness?  just a thought.

Reply:
This just follows the existing logic, but yea we can always try to improve it.